app/blog/models.py

```diff
@@ -25,6 +25,9 @@ def save(self, *args, **kwargs):
             self.slug = slugify(self.title)
         return super().save(*args, **kwargs)
 
+    class Meta:
+        ordering = ["-created_at"]
```

Comment:
ordering adds quite a performance overhead in all your queries. If you are sure you are always going to need this ordering, this is fine. Otherwise I would just specify it in your queries whenever you need it.

Reply:
Pagination may yield inconsistent results with an unordered object_list